src/map.config.js

```diff
@@ -42,7 +42,7 @@ const tms = true
 const tileLayers = [
   {
     title: 'Achtergrondkaart',
-    url: 'http://geodata.nationaalgeoregister.nl/tms/1.0.0/brtachtergrondkaart/{z}/{x}/{y}.png'
+    url: 'https://geodata.nationaalgeoregister.nl/tms/1.0.0/brtachtergrondkaart/{z}/{x}/{y}.png'
```

Comment:
Can't we make this a protocol-less url? ie `//geodata.nationaalgeoregister.nl/` ?

Reply:
Oh this is considered an antipattern as of 2014, my bad!
https://www.paulirish.com/2010/the-protocol-relative-url/